docs/pages/develop/user-interface/splash-screen-and-app-icon.mdx

```diff
@@ -4,153 +4,123 @@ description: Learn how to add a splash screen and app icon to your Expo project.
 ---
 
 import { Collapsible } from '~/ui/components/Collapsible';
-import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
```

Comment:
Since the static images are not used in this new version of the guide. Can you remove all the static resources that were previously used from `public/static/images/splash-screen`?

Reply:
Oh yes! I forgot! 😅